src/esssans/loki/io.py

```diff
@@ -38,6 +38,8 @@ def _patch_data(
 ) -> sc.DataArray:
     out = da.copy(deep=False)
     if out.bins is not None:
+        # Currently ScippNexus adds 32 bit event weights, this may cause trouble.
+        out.bins.data = out.bins.data.to(dtype='float64', copy=False)
```

Comment:
Does this also happen with the ISIS files?

Reply:
I think so actually. We should probably have a chat about this. It would be nice if we could keep 32 bit until needed, so I was not happy about changing this here.

Reply:
I reverted this change (and fixed the tests instead). Previously there was an (maybe unintentional) conversion done in `normalize`, which this PR remove.

I opened #88 instead.

Can you have another brief look @nvaytet?